templates/github/.github/workflows/scripts/update_ci.sh.j2

```diff
@@ -9,29 +9,15 @@ if [ ! -f "template_config.yml" ]; then
 fi
 
 pushd ../plugin_template
-./plugin-template --github {{ plugin_name }}
+./plugin-template --github {% if ci_update_docs -%} --docs {% endif -%} {{ plugin_name }}
 popd
 
 # Check if only gitref file has changed, so no effect on CI workflows.
-if [[ $(git diff --name-only) == ".github/template_gitref" ]]; then
-  echo "No changes detected in github section."
-  git restore ".github/template_gitref"
+if [ "$(git diff --name-only | grep -v "template_gitref")" ]; then
+  echo "No changes detected."
+  git restore ".github/template_gitref" "docs/template_gitref"
```

Comment:
Maybe this line should just be `git restore "."` or can be we be more specific and restore anything named "template_gitref"?